spec/omniauth/auth_hash_spec.rb

```diff
@@ -8,6 +8,11 @@
     expect(subject.info.first_name).to eq('Awesome')
   end
 
+  it 'does not try to parse `string` as InfoHash' do
```

Comment:
I'm not convinced that this test covers your use-case as it's technically already tested here:

https://github.com/estepan/omniauth/blob/ed89156612311e0d2ced47c73ea7d0748e5a7eef/spec/omniauth/auth_hash_spec.rb#L5-L9